### PR TITLE
Pyang

### PR DESCRIPTION
**Pull request by ksator**
add pyang examples

### CODE REVIEW

File: docs/examples/pyang/index.md [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+```shell
+pip install pyang
+pip3 freeze | grep pyang
+```
+
+```shell
+pip3 freeze | grep pyang
+```
+
+## we need yang modules
```
Comment (fredhsu) [inline docs/examples/pyang/index.md:29]: Is this supposed to be a H2?

File: docs/examples/pyang/index.md [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+
+### Create a directory
+
+```shell
+mkdir yang_modules
+```
+
+### Clone the openconfig repository
```
Comment (fredhsu) [inline docs/examples/pyang/index.md:37]: For consistency use OpenConfig instead of openconfig

File: docs/examples/pyang/index.md (hunk 1 above)
Comment (ksator) [reply docs/examples/pyang/index.md:29]: done, thanks

File: docs/examples/pyang/index.md (hunk 2 above)
Comment (ksator) [reply docs/examples/pyang/index.md:37]: done, thanks

**Review by fredhsu — APPROVED** [review]
lgtm

**Pull request merged**